user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Tutorials/Incident_Tracking_Configuration_Tutorial.md

```diff
@@ -0,0 +1,248 @@
+        </item>
+      ```
+
+      The result should look like this:
+
+      ![Grouping on custom property with pasted snippet](~/user-guide/images/customPropertyGrouping2.png)
+
+   1. In the *name* tag of the pasted snippet, fill in the property name on which you want to group, i.e. *ParentNode*.
+
+   1. Save the file.
+
+   1. Restart *SLAnalytics.exe* (e.g. through the task manager).
+
+   > [!NOTE]
+   > If you do not have access to the *configuration.xml* file, instead run the Automation script *Incident Tracking Tutorial Scripts - Script 2 Enable Custom Alarm Property Grouping* to apply the same configuration.
+
+   > [!TIP]
+   > For more detailed information about adding custom rules, see [Defining custom properties for grouping](xref:Customizing_alarm_grouping_rules#defining-custom-properties-for-grouping).
+
+1. On the amplifier elements ending in *Node1*, go to the *Forward Path* > *Level Detector* page and click *Toggle Alarm* to generate alarms.
+
+1. In the Alarm Console, check whether an alarm group has been created.
+
+   The resulting alarm group should look like this:
+
+   ![Grouping on alarm property result](~/user-guide/images/GroupingOnAlarmPropertyResult.png)
+
+1. Clear the created alarms by clicking the *Toggle Alarm* buttons again.
+
+## Step 6: Group on a custom element property
+
+In this step, you will create a rule to group alarms on amplifiers under the same node, but only if **at least 75%** of the amplifiers under the node are in alarm.
+
+You should find everything you need to complete this exercise on the [Customizing alarm grouping rules](xref:Customizing_alarm_grouping_rules) page.
+
+To help you on your way, we recommend creating a custom element property. For each amplifier, the value of this property should refer to its parent node.
+
+In case you do not have access to the *configuration.xml* file and you would still like to participate:
+
+1. Open the *Incident Tracking Tutorial Scripts - Script 3 Enable Custom Element Property Grouping* automation script.
+1. Look for the string variable called *KataExercise*.
+1. Set this string equal to all the xml code that you want to add to the *value* tag of the *GenericProperties* rule in the *configuration.xml*.
+1. Save the file.
+1. Restart *SLAnalytics.exe* through the task manager.
+1. Send us a screenshot to be awarded DevOps Points.
+
+   > [!IMPORTANT]
+   > Use the following email format to send us your screenshot:
+   >
+   > - Subject: Tutorial - Fine-tune incident tracking in your system
+   > - To: [ai@skyline.be](mailto:ai@skyline.be)
+   > - Body:
+   >   - Dojo account: Clearly mention the email address you use to sign into your Dojo account, especially if you are using a different email address to send this email.
+   >   - Attachment: Add a screenshot of the amplifiers you have set in alarm and of the corresponding alarm group.
+   >
+   > Upon successful validation, you will be awarded 75 DevOps Points.
```

Comment:
Could you adjust this step so that it's more clear what exactly the user should do? Right now, you're kind of dropping them into the deep end and having them figure things out themselves, but this isn't really what we want with these tutorials. As a user following a tutorial, I wouldn't want to be forced to go read another section of the help to be able to continue; instead I'd want to have the basic steps, with a link to another section in case I want to dive deeper (the way I did it for step 5). Could you also clarify why it's necessary to add another custom element property, instead of using the *parentNode* property from the previous step?